nautobot_golden_config/tables.py

```diff
@@ -467,11 +499,31 @@ class ConfigPlanTable(StatusTableMixin, BaseTable):
     feature = LinkColumn(
         "plugins:nautobot_golden_config:compliancefeature", args=[A("feature__pk")], text=lambda record: record.feature
     )
+    config_set = TemplateColumn(template_code=CONFIG_SET_BUTTON, verbose_name="Config Sets", orderable=False)
```

Comment:
```suggestion
    config_set = TemplateColumn(template_code=CONFIG_SET_BUTTON, verbose_name="Config Set", orderable=False)
```

Singular for consistency.